seven/voice.go

```diff
@@ -18,7 +18,7 @@ type Voice struct {
 	Balance    float64        `json:"balance"`
 	Debug      bool           `json:"debug"`
 	TotalPrice float64        `json:"total_price"`
-	Success    string         `json:"success"`
+	Success    StatusCode     `json:"success"`
```

Comment:
The return codes for the voice call endpoint are not documented, so I assume that they are the same as for the SMS endpoint?